packages/wcs-lib/wcs-lib.2017-05-26.02/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "1.2"
+name: "wcs-lib"
+version: "2017-05-26.02"
+maintainer: [
+  "Louis Mandel <lmandel@us.ibm.com>"
+]
+authors: [
+  "Guillaume Baudart"
+  "Louis Mandel"
+  "Jérôme Siméon"
+]
+homepage: "https://github.com/IBM/wcs-ocaml"
+bug-reports: "https://github.com/IBM/wcs-ocaml/issues"
+license: "Apache License 2.0"
+dev-repo: "https://github.com/IBM/wcs-ocaml.git"
+
+build: [
+  ["jbuilder" "subst" "-p" name "--name" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+build-test: [ ["jbuilder" "runtest"] ]
+
+depends: [
+  "ocamlfind"
```

Comment:
If you're using jbuilder, it's unlikely that ocamlfind is a direct dependency of yours. Hence it should be removed.